hr_timesheet_overtime/models/hr_employee.py

```diff
@@ -24,11 +22,13 @@ class HrEmployee(models.Model):
         string="Initial Overtime",
         default=0.0,
         help="Initial Overtime to start Overtime Start Date with",
+        groups="hr.group_hr_user",
     )
     total_overtime = fields.Float(
         string="Total Overtime",
         compute="_compute_total_overtime",
         help="Total Overtime since Overtime Start Date",
+        groups="hr.group_hr_user",
```

Comment:
I'm still confused by how Odoo permissions work, but I think this line is undesireable. It means that users (who are not HR) cannot view their own overtime.

Reply:
you’re right! fixed! i got confused during my tests: related fields ignore model access rights, but not field access rights, apparently. thanks and sorry for the confusion caused on #16.

Reply:
Thanks!